src/python/test_LossFunctionTranslation.py

```diff
@@ -0,0 +1,127 @@
+        loss = LossFunctionTranslation().to_loss_function(resources, json_dict)(context)
+        self.assertEqual(loss.shape, (1,))
+        self.assertEqual(loss, 5)
+
+
+    def test_negation(self):
+        resources = {}
+        json_dict = self.load_json('test_negation')
+        context = []
+        loss = LossFunctionTranslation().to_loss_function(resources, json_dict)(context)
+        self.assertEqual(loss, -1)
+
+
+    def test_minimum(self):
+        resources = {}
+        json_dict = self.load_json('test_minimum')
+        context = []
+        loss = LossFunctionTranslation().to_loss_function(resources, json_dict)(context)
+        self.assertEqual(loss, 7)
+
+
+    def test_maximum(self):
+        resources = {}
+        json_dict = self.load_json('test_maximum')
+        context = []
+        loss = LossFunctionTranslation().to_loss_function(resources, json_dict)(context)
+        self.assertEqual(loss, 12)
+
+
+    def test_subtraction(self):
+        resources = {}
+        json_dict = self.load_json('test_subtraction')
+        context = []
+        loss = LossFunctionTranslation().to_loss_function(resources, json_dict)(context)
+        self.assertEqual(loss, 5)
+
+
+    def test_indicator(self):
+        resources = {}
+        json_dict = self.load_json('test_indicator')
+        context = []
+        loss_not_equal = LossFunctionTranslation().to_loss_function(resources, json_dict[0])(context)
+        loss_equal = LossFunctionTranslation().to_loss_function(resources, json_dict[1])(context)
+        self.assertEqual(loss_not_equal, 0)
+        self.assertEqual(loss_equal, 1)
+
+
+    def test_at(self):
+        resources = {}
+        json_dict = self.load_json('test_at')
+        context = [[4, 13, 22, 1]]
+        loss = LossFunctionTranslation().to_loss_function(resources, json_dict)(context)
+        self.assertEqual(loss, 22)
+
+
+    def test_network(self):
+        model = keras.Sequential([
+            keras.layers.Input(shape=(1,)),
+            keras.layers.Dense(units=1, kernel_initializer='ones'),
+        ])
+        resources = {'f': model}
+        json_dict = self.load_json('test_network')
+        context = [2, 3.5, 14]
+        loss = LossFunctionTranslation().to_loss_function(resources, json_dict)(context)
+        self.assertEqual(loss, 3.5)
+
+
+    def test_quantifier(self):
+        resources = {}
+        json_dict = self.load_json('test_quantifier')
+        context = []
+        loss_all = LossFunctionTranslation().to_loss_function(resources, json_dict[0])(context)
+        loss_any = LossFunctionTranslation().to_loss_function(resources, json_dict[1])(context)
+        self.assertEqual(loss_all, 50)
+        self.assertEqual(loss_any, 12)
+
+    
+    def test_lambda(self):
+        resources = {}
+        json_dict = self.load_json('test_lambda')
+        context = []
+        loss = LossFunctionTranslation().to_loss_function(resources, json_dict)(context)(3)
+        self.assertEqual(loss, 3)
```

Comment:
Should add a few more values here to test that it really is encoding a function rather than simply the constant `3`.